scipy/stats/tests/test_distributions.py

```diff
@@ -1064,6 +1064,27 @@ class TestTruncnorm:
     def setup_method(self):
         np.random.seed(1234)
 
+    @pytest.mark.parametrize("a, b, ref",
+    [(1, 1.1, -2.3030441048144877733524473299116314416365215401594087),
+    (0.6, 0.7, -2.3027610681852571124367653023002056177369149733744606),
+    (1, 3, 0.29622333134332169527554642294689301750721849032204733)])
+    def test_entropy(self, a, b, ref):
+        #All reference values were calculated with mpmath:
+        #def entropy_trun(a, b):
+        #    def cdf(x):
+        #        return 0.5 * (1 + mp.erf(x / mp.sqrt(2)))
+        #
+        #    Z = cdf(b) - cdf(a)
+        #
+        #    def pdf_standard_norm(x):
+        #        return (mp.exp(-0.5 * x ** 2) / mp.sqrt(2 * mp.pi))
```

Comment:
mpmath has functions available for the standard normal pdf and cdf: https://mpmath.org/doc/current/functions/expintegrals.html#npdf

They could save you some work here.

Reply:
The code was already run, so it doesn't matter. Thanks for the heads up, though.